boat-scaffold/src/main/java/com/backbase/oss/codegen/angular/BoatAngularGenerator.java

```diff
@@ -69,6 +70,7 @@ public class BoatAngularGenerator extends AbstractTypeScriptClientCodegen {
     public static final String HAS_EXAMPLES = "hasExamples";
     public static final String PATTERN = "pattern";
     protected String foundationVersion = "6.6.7";
+    protected String specVersion = "1.0.0";
```

Comment:
I'm not sure it's a good idea to default to a particular version; it will create incorrect results for people who are not setting this property.